components/ChallengeMaterial.tsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react'
+import ChallengeTitleCard from '../components/ChallengeTitleCard'
+import ChallengeQuestionCard from '../components/ChallengeQuestionCard'
+
+const ChallengeMaterial: React.FC = () => {
+  return (
+    <div className="row col-12">
```

Comment:
If you removed `col-12` borders will be align like the design
<img width="1190" alt="Screen Shot 2020-03-08 at 10 25 51 AM" src="https://user-images.githubusercontent.com/25457563/76167947-c0885380-6127-11ea-8634-4b9fc38a721f.png">